x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/BlobCacheMetrics.java

```diff
@@ -33,14 +54,39 @@ public BlobCacheMetrics(MeterRegistry meterRegistry) {
                 "es.blob_cache.cache_miss_load_times.histogram",
                 "The time in milliseconds for populating entries in the blob store resulting from a cache miss, expressed as a histogram.",
                 "ms"
+            ),
+            meterRegistry.registerDoubleHistogram(
+                "es.blob_cache.populate_throughput.histogram",
+                "The throughput when populating the blob store from the cache",
+                "MB/second"
+            ),
+            meterRegistry.registerLongCounter(
+                "es.blob_cache.populate_bytes.total",
+                "The number of bytes that have been loaded into the cache",
+                "bytes"
+            ),
+            meterRegistry.registerLongCounter(
+                "es.blob_cache.populate_time.total",
+                "The time spent copying data into the cache",
+                "milliseconds"
```

Comment:
Maybe I missed the discussion somewhere: I thought this should be a histogram similar to s3 http request time?

Reply:
We did discuss this, the feeling was that because we've got the throughput distribution, it might give us more flexibility to record population bytes and time as raw totals. Leaving them as raw totals leaves more options for aggregation in the charts (e.g. how much did we download when warming shard X, how long did we spend warming index Y, how much did we download due to warming when that node failed) I don't think you can answer those questions with bytes/time histograms.